odh-manifests/smaug/kafka/base/kafka-cluster.yaml

```diff
@@ -1,75 +0,0 @@
----
```

Comment:
@margarethaley along with this, you should be able to delete the [kafka kfdefs directory](https://github.com/operate-first/apps/tree/master/kfdefs/base/kafka), just move the [kafka pod monitors](https://github.com/operate-first/apps/blob/master/kfdefs/base/kafka/kafka-podmonitors.yaml) to the new kafka smaug overlay.

Reply:
@4n4nd why are we deleting this? Shouldn't this be moved to the `kafka/overlays/smaug/` location?

Reply:
@HumairAK we already have a kafka resource in the kafka [base](https://github.com/operate-first/apps/blob/master/kafka/base/opf-kafka.yaml)